HOMEWORK.js

```diff
@@ -40,15 +40,15 @@ function homework (renderList) {
 	// ◽ bonus: do not allow new items to be less than 3 characters long
 	function onAddTodo (newTodoText) {
 		// Add code here
-
+		newTodoText.length > 3 ? todos.push(newTodoText) : alert("Please enter at last 3 characters");
```

Comment:
I like that you've made us of the alert function from later on in the exercise to solve this problem too.

Reply:
However, normally we would use ternary operators only if we are returning a value, like this:
```js
var x = (y > 10 ? "big" : "small");
```
When we are doing complex logic, we normally use `if/else` because it is often easier to read.